docs/design_guide.rst

```diff
@@ -383,6 +406,14 @@ Renders as:
 
   :param float degrees: Degrees to turn right
 
+Documentation References to other Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you need to make references to documentation in other libraries you should refer the class using single
+backticks  ``:class:`~adafruit_motor.servo.Servo```. You must add also the reference in the ``conf.py`` file in the
```

Comment:
Thank you for adding this! 

@jposada202020 You could update this to read "`You must also add`". It flows a little better.

Reply:
No problem, will do,

Reply:
Changed in https://github.com/adafruit/circuitpython/pull/4652/commits/a512dd230df419c204b7acd49c0cc7f9ebdce290